extensions/curator/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM alpine:3.8
 
 RUN apk --update add --no-cache python py-pip \
-  && pip install elasticsearch-curator
+  && pip install elasticsearch-curator==5.5.4
```

Comment:
Excellent idea. Would you mind setting the version using an environment variable to make it clearer? [example](https://github.com/deviantony/docker-elk/blob/searchguard/elasticsearch/Dockerfile#L8-L10)